earthpy/tests/test_plot_bands.py

```diff
@@ -17,6 +17,11 @@ def one_band_3dims():
     )
 
 
+@pytest.fixture
+def one_band_3dims():
```

Comment:
How did this get in here?

Reply:
@mbjoseph i was cleaning up duplicate fixtures and noticed issues with conftest. i then realized there were still fixtures in conftest that should be in a specific test module. so i moved it. do you want me to move this over to another pr? it's such a minor change and it was part of my larger clean up all tests effort in this work ... so that is what happened!!

i'll work on the other edits over the next few days!! thank you so much for these.

Reply:
@lwasser I think this is a duplicate fixture that has yet to be cleaned up. It already exists on the master branch in this file, and on this branch the fixture `one_band_3dims` is defined twice now in `test_plot_bands.py`.

Reply:
oh you're right. this is my old version of this fixture. i'm not sure how all of these fixtures got duplicated... actually i have a suspicion ...